### PR TITLE
[SYCL][Matrix] Disable test Legacy/element_wise_all_ops_int8_packed.cpp for cpu

### PR DESCRIPTION
**Pull request by yubingex007-a11y**

### CODE REVIEW

File: sycl/test-e2e/Matrix/Legacy/element_wise_all_ops_int8_packed.cpp [hunk 1]
```diff
@@ -10,7 +10,7 @@
 // RUN: %{build} -o %t.out -DSYCL_EXT_ONEAPI_MATRIX_VERSION=1
 // RUN: %{run} %t.out
 
-// XFAIL: gpu
+// XFAIL: *
```
Comment (YuriPlyakhin) [inline sycl/test-e2e/Matrix/Legacy/element_wise_all_ops_int8_packed.cpp:13]: The test should pass on GPU with updated IGC. Should it be like that instead?
```suggestion
// XFAIL: cpu
```

**Review by YuriPlyakhin — CHANGES_REQUESTED** [review]

**Comment by YuriPlyakhin** [comment]
I don't know, if it should pass or fail on CPU. @dkhaldi , could you please confirm?

**Comment by dkhaldi** [comment]
> I don't know, if it should pass or fail on CPU. @dkhaldi , could you please confirm?

If you look at https://github.com/intel/llvm/blob/sycl/sycl/test-e2e/Matrix/Legacy/element_wise_all_ops_int8_packed_impl.hpp
you will find statements like:
` joint_matrix_store(sg, sub_b,
                              accA.get_pointer() + (sg_startx * TM) * N * 4 +
                                  sg_starty / SG_SZ * TN * 4,
                              N * 4, matrix_layout::row_major);`

This is a store from sub_b that is in VNNI format to memory in row major. Currently, we dont support such store. The only store we can do is a store with packed as matrix layout.

That's why this test cannot pass on SPR (or on the GPU for that matter as the same limitation exist currently for the GPU implementation)

**Comment by aelovikov-intel** [comment]
@dkhaldi , does it mean that you agree with this patch? If so, can you please formally approve it?

**Review by dkhaldi — APPROVED** [review]
LGTM

**Review by aelovikov-intel — APPROVED** [review]

**Pull request merged**